api/server.spec.js

```diff
@@ -6,8 +6,10 @@ describe('/ [GET]', () => {
     request(server)
       .get('/')
       .expect('Content-Type', /json/)
-      .expect(200);
-      done
+      .then(res =>{
```

Comment:
Add a space between the `=>` and the `{`. 
Configure and use prettier to ease the process.

Reply:
Done. Prettier has been used

1.  prettier extensions settings enabled
2. prettier formatted